webapp/templates/standalone.html

```diff
@@ -0,0 +1,19 @@
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+    <meta name="viewport" content="width=device-width" />
+    <title>Pyroscope</title>
+
+    <script type="text/javascript">
+      try {
+        eval('window.flamegraph = {{ .Flamegraph }}');
+      } catch(e) {}
+    </script>
+  </head>
+  <body>
+    <div id="root"></div>
```

Comment:
Shall we add a pixel tracker? @petethepig @Rperry2174

Reply:
@eh-am that would be too invasive imo. we could add a new metric to analytics module, something like `html_exports`, and update the counter every time someone exports an html

Reply:
I was hoping to somehow know if people are **sharing** the export HTML around, but fair point, it's very intrusive yes.